frontend/components/Players/PlayerManager.vue

```diff
@@ -279,28 +278,19 @@ export default Vue.extend({
   },
   mounted() {
     document.addEventListener('mousemove', this.handleMouseMove);
+    window.addEventListener('keyup', this.handleKeyPress);
+    window.addEventListener('click', this.handleVideoClick);
 
     this.addMediaHandlers();
-
-    this.unsubscribe = this.$store.subscribe((mutation, state: AppState) => {
-      switch (mutation.type) {
-        case 'playbackManager/TOGGLE_MINIMIZE':
-          if (state.playbackManager.isMinimized === true) {
-            window.removeEventListener('keydown', this.handleKeyPress);
-          } else if (state.playbackManager.isMinimized === false) {
-            window.addEventListener('keydown', this.handleKeyPress);
-          }
-
-          break;
-      }
-    });
```

Comment:
How you handle this now?

This means that clicking in the button directly (without a keypress) doesn't perform the action, which may be conflicting for music for instance

Reply:
I'm not on the project right now, but this is the mutation subscription, right?

Shouldn't it be done in every case where mutation occurs? (therefore also pressing with a "physical" button instead of keyboard)

Reply:
The unsubscription doesn't happen on every mutation, just when the component is unmounted. 

With the subscription you make sure that any mutation to the store that changes the mutation store is handled. Without it, you can whatever you want that nothing will happen.

Reply:
Sorry, I didn't realize it was code I removed.
----
If I removed it, it is because the listener is used for the player mini as well.

If you notice, the listener code, performs the actions based on whether the player is mini or normal, so the listener can always remain active, instead of being inserted and removed (this mainly because "f" for the player mini, puts it back to normal player, as the description of the PR indicates).

Reply:
@doc-code-hub But how other components dispatching the mutation works now then? You must put a replacement in place then.

Reply:
I see no problems on the horizon, considering that a check on isMinimized takes place in the handler (which is a property mapped directly from the "playbackManager" store).

So if another component were to change the isMinimized state, this would be automatically updated in the component as well, making sure that the handler always has the current and reliable value of "isMinimized".

It shouldn't matter who throws an "isMinimized" mutation since it is a property mapped to the component with mapState.

Do you agree @ferferga?

Reply:
Fine, missed the mapped state, sorry!